docs/ProductionChecklistForScalarDBCluster.md

```diff
@@ -0,0 +1,113 @@
+# Production checklist for ScalarDB Cluster
+
+This checklist provides recommendations when deploying ScalarDB Cluster in a production environment.
+
+## Before you begin
+
+In this checklist, we assume that you are deploying ScalarDB Cluster on a managed Kubernetes cluster, which is recommended.
+
+## Production checklist: ScalarDB Cluster
+
+The following is a checklist of recommendations when setting up ScalarDB Cluster in a production environment.
+
+### Number of pods and Kubernetes worker nodes
+
+To ensure that the Kubernetes cluster has high availability, you should use at least three worker nodes in three availability zones and deploy at least three pods.
+
+### Worker node specifications
+
+From the perspective of commercial licenses, resources for one pod running ScalarDB Cluster are limited to 2vCPU / 4GB memory. In addition, there are some pods other than ScalarDB Cluster pods on the worker nodes.
+
+In other words, the following components run on one worker node:
```

Comment:
```suggestion
In other words, the following components could run on one worker node:
```
I think running application pods in the same worker node depends on users' preferences/requirements. 
So, stating as above might sound like we assume applications' pods should be deployed in the same worker node.
What do you think?

@josh-wong Also, how does it sound to you?

Reply:
That makes sense to me👍